test_projects/test_project_e2e_with_suffix/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+set(theseTestsName test_project_e2e)
```

Comment:
>test_project_e2e [](http://example.com/codeflow?start=19&length=16)

I don't understand why this project is called test_project_e2e and the folder is called test_project_e2e_with_suffix?  I don't see any suffix.  What am I missing?

Reply:
You're not missing anything, this project was initially added when I had in mind to have the suffix as an option. Afterwards I decided we can simply use the project name, thus, this project is no longer needed.

Removed in the last iteration.